src/runners/flow-web.js

```diff
@@ -1,7 +1,8 @@
 import SwaggerParser from '../parsers/swagger/Parser'
 import RAMLParser from '../parsers/raml/Parser'
 import PostmanParser from '../parsers/postman/Parser'
-import CurlParser from '../parsers/curl/Parser'
+import CurlParser from '../parsers/cURL/Parser'
```

Comment:
`curl => cURL` In theory, this should not make any difference, as node and babel ignore case, but webpack cares about it, and it's better anyway